vaadin/src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+#Vaadin supports spring-boot 2.1 properly from V8 onwards (according to this comment https://github.com/vaadin/spring/issues/331#issuecomment-435128475)
+spring.main.allow-bean-definition-overriding=true
```

Comment:
As I mentioned in the notes, vaadin supports spring 2.1 properly from version 8 onwards, that's why I decided to 'bypass' a bean overriding exception (caused by spring and vaadin autoconfigurations, nothing that we can handle) using this property